src/settings/instructions.ts

```diff
@@ -0,0 +1,50 @@
+export interface IDict<T = any> {
+  [key: string]: T;
+}
+
+const chromeAiInstructions = `
```

Comment:
Probably fine to leave them here in one file here for now.

Later when third-party extensions can register additional AI providers (https://github.com/jupyterlite/ai/issues/45) they could also provide their own instructions for the provider.